org-code-javabuilder/lib/src/main/java/org/code/javabuilder/JavaRunner.java

```diff
@@ -43,22 +43,28 @@ public JavaRunner(
    *     finished executing.
    */
   public void runMain() throws InternalFacingException, JavabuilderException {
-    this.run(this.mainRunner);
+    this.run(this.mainRunner, false);
   }
 
   public void runTests() throws JavabuilderException, InternalFacingException {
-    this.run(this.testRunner);
+    // Tests have more permissions than a regular run--as of now, all
+    // tests have access to the org.code.validation package.
+    this.run(this.testRunner, true);
   }
 
-  private void run(CodeRunner runner) throws JavabuilderException, InternalFacingException {
+  private void run(CodeRunner runner, boolean hasElevatedPermissions)
```

Comment:
Nice solution! I wonder, if there's a chance we may have a more complicated permissions model down the road, would it be worth making this an enum? i.e. PermissionLevel.USER, PermissionLevel.VALIDATOR/TESTER or something like that?

Reply:
good call, we will probably want user, validator and tester down the line so I'll update this (for now just with user and tester, we can add validator later once we are running validation code separate from project test code)